test/edu/csus/ecs/pc2/imports/ccs/ContestSnakeYAMLLoaderTest.java

```diff
@@ -3500,6 +3501,118 @@ public void testScoreboardHTMLLocations() throws Exception {
 
     }
     
+    public void testLoadDisplay() throws Exception {
+        
+        String dirname = getOutputDataDirectory(getName());
+        ensureDirectory(dirname);
+        
+        String[] yamlLines = { //
+                "",
+                "team-display-on-scoreboard-string : '{:teamname}'", //
+                "",
+                "accounts:", //
+                "  - account: TEAM", //
+                "    site: 1", //
+                "    count: 22", //
+        };
+
+
+        IInternalContest contest = loader.fromYaml(null, yamlLines, dirname);
+        
+        Account[] teams = getTeamAccounts(contest);
+        assertEquals("Team count", 22, teams.length);
+
+        String displayString = contest.getContestInformation().getTeamDisplayOnScoreboard();
+        assertEquals("Team getTeamDisplayOnScoreboard", "{:teamname}", displayString);
+        
+        /**
+         * 
+     * Client/Team number - {:clientnumber} = 514
+     * Country Code - {:countrycode} = CAN
+     * CMS/External ID - {:externalid} = 309407
+     * Group Id Number - {:groupid} = 12545
+     * Group name - {:groupname} = Canada - University of British Columbia D1
+     * Long School name - {:longschoolname} = UBC! (U British Columbia)
+     * Short Shool name - {:shortschoolname} = U British Columbia
+     * Team site number - {:sitenumber} = 1
+     * Team login name - {:teamloginname} = team514
+     * Team name - {:teamname} = UBC!
+         */
+
+        Account account20 = getSortedTeamAccounts(contest)[20];
```

Comment:
This variable name is a bit misleading; upon completion of this statement, the variable named `account20` actually holds the value for account 21.  I would suggest naming the variable `account21`.  That would be consistent with the naming you used in `ScoreboardVariableReplacerTest`...

Reply:
no change.

Reply:
I respect your right to decline to make a change.  I'll just point out (again) that this code is using a pattern which is identical to that used in `ScoreboardVariableReplacerTest` -- but it's using a different convention for applying that pattern.  Seems confusing to readers...